openassessment/fileupload/backends/s3.py

```diff
@@ -8,6 +8,8 @@
 import botocore
 import boto3
 
+from botocore.client import Config
```

Comment:
nit: any reason not to use the already-imported botocore library above?
Then below, it would just be

```
config=botocore.client.Config(
...
```

Reply:
It was just looking cleaner :) and there was just one occurrence, I have corrected that now. :)